### DISCUSSION TITLE
FRT Self Learning Sessions Problems

### DISCUSSION THREAD

**Discussion by iamshivampakade** [github/india#327] [Open Source]
I Have completed my first module, but its is still showing in progress at FRT Self Learning Platform!

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by FRTMentor** [comment]
It will take minimum 24 hours to reflect on the application.
If you're still facing the same issue then please reach out our support team.
support@futurereadytalent.in
